uint32 loc_id
---
